docs/extensions/guides/renderer-extension.md

```diff
@@ -570,15 +570,17 @@ export default class HelpExtension extends LensRendererExtension {
 
   statusBarItems = [
     {
-      item: (
-        <div
-          className="flex align-center gaps"
-          onClick={() => this.navigate("help")}
-        >
-          <HelpIcon />
-          My Status Bar Item
-        </div>
-      ),
+      components: {
```

Comment:
@jakolehm, are the public docs for 4.0 pulled from master or a release branch? I think @pauljwil is making changes to docs (which would be good for 4.0.x and above) in master, but the change in this PR is strictly 4.1

If #1789 is meant for 4.1 only then there's no issue. If we want #1789 to be reflected in 4.0.x docs then we have to be careful here

Reply:
Docs are generated from a release.